### DISCUSSION TITLE
Clarification on government payer reimbursements

### DISCUSSION THREAD

**Discussion by JaSimps5** [CMSgov/hospital-price-transparency#59] [Q&A] [answered]
For payers such as Medicare or Tricare and all managed versions of these,  the negotiated rates are generally a percentage of the government calculation.  Specifically for DRGs, those calculations are complex and depend on many variables.  Can you please clarify if a negotiated payer reimburses at 100% of Medicare or Tricare for Inpatient, how is that to be shown on the file?  

I have asked around to several colleagues and in total there were three separate assumptions for the Medicare and Tricare DRGs:
1. this is what the case rate example in Rows 4-6, Platform Health Insurance PPO is showing, but besides the algorithm piece that mentions MS-DRG, this is just a guess.  
2. Since these are variable this should be shown like Row 7, Region Health Insurance HMO, putting the percentage in percent field and a payer note indicating this is a percentage of Medicare or Tricare.   
3. this should be listed as an algorithm 

As you can see, there is definitely distinct differences in understanding of these rules, so any further clarification you can provide would be incredibly helpful.

**Comment by RidenourK** [comment, marked as answer]
@JaSimps5. Thank you for your question. Please refer to the [Hospital Price Transparency FAQs](https://www.cms.gov/files/document/hospital-price-transparency-frequently-asked-questions.pdf) on the [CMS HPT Resources Page](https://www.cms.gov/priorities/key-initiatives/hospital-price-transparency/resources) where this question is addressed.
